plugins/PaintTool/PaintView.py

```diff
@@ -114,14 +116,17 @@ def _makePaintModes(self):
     def _makeExtrudersColors(self) -> Dict[str, "PaintView.PaintType"]:
         extruders_colors: Dict[str, "PaintView.PaintType"] = {}
 
-        for extruder_item in self._extruders_model.items:
-            if "color" in extruder_item:
+        for extruder_index in range(PaintView.MAX_EXTRUDER_COUNT):
+            extruder_item = self._extruders_model.getExtruderItem(extruder_index)
+            if extruder_item is None:
+                extruder_item = self._extruders_model.getExtruderItem(0)
+
+            if extruder_item is not None and "color" in extruder_item:
```

Comment:
I personally find this a bit cleaner since I read the second if-statement as a continuation of the first (and it saves one comparison)
```suggestion
            elif "color" in extruder_item:
```